main/main.cpp

```diff
@@ -1941,8 +1950,7 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 }
 
 Error Main::setup2(Thread::ID p_main_tid_override) {
-	// Print engine name and version
-	print_line(String(VERSION_NAME) + " v" + get_full_version_string() + " - " + String(VERSION_WEBSITE));
+	print_engine_name_and_version();
```

Comment:
I also extracted this into a helper function since it is used twice in main.cpp.